test/runtests.jl

```diff
@@ -210,7 +211,7 @@ end
 
     if Cairo.libcairo_version >= v"1.12.0"
 
-        if ~is_windows()
+        if ~Sys.iswindows()
```

Comment:
Yeah, that's a failure of the documentation.
```
julia> using Compat.Sys

julia> Sys.iswindows()
WARNING: both Sys and Base export "Sys"; uses of it in module Main must be qualified
ERROR: UndefVarError: Sys not defined
```
Though actually it appears to work (at least on 0.6) with `import Compat.Sys`.